sign/ed448/ed448.go

```diff
@@ -96,7 +96,10 @@ func NewKeyFromSeed(seed PrivateKey) *KeyPair {
 
 	pair := &KeyPair{}
 	copy(pair.private[:], seed)
-	_ = goldilocks.Curve{}.ScalarBaseMult(s).ToBytes(pair.public[:])
+	P := &goldilocks.Point{}
+	goldilocks.Curve{}.ScalarBaseMult(P, s)
+	enc, _ := P.MarshalBinary()
```

Comment:
Error being ignored intentionally?

Reply:
Encoding always succeeds, but Go's standard MarshalBinary interface doesn't care.  Maybe it's worthwhile having a separate `PackInto` function that writes into a buffer instead of returning it.  Saves an allocation and then you don't have an error to confuse people about.

Reply:
Since this error will not occur, added a panic if it happens. Not the best way to handle this error.
I will consider adding PackInto.